src/main/resources/tags/util/logsdump.tag

```diff
@@ -0,0 +1,8 @@
+type: text
+image: https://media.discordapp.net/attachments/774372288061243482/776957132465373184/97361099-751ce400-1875-11eb-84b5-2945d840dfff.png
+
+---
+
+Run `geyser dump logs` (server console) / `/geyser dump logs` (ingame chat) on your Geyser instance or server, then copy the link it will give you and paste it here. This will auto upload the server logs to mclo.gs and provide its url into the dump.
```

Comment:
```suggestion
Run `geyser dump logs` (server console) / `/geyser dump logs` (ingame chat) on your Geyser instance or server, then copy the link it will give you and paste it here. This will automatically upload the server logs to [mclo.gs](https://mclo.gs/) and add the link in the dump.
```